cmake/std/PullRequestLinuxDriver.sh

```diff
@@ -25,6 +25,97 @@ function get_md5sum() {
     echo "${sig:?}"
 }
 
+#
+# Get pip
+# - @param1 python_exe - the python executable to install PIP for
+function get_pip() {
+    local python_exe=${1:?}
+
+    echo -e "--- Python: ${python_exe:?}"
+
+    # fetch get-pip.py
+    echo -e "--- curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py"
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```

Comment:
Do you want to use a pattern like this to avoid potential glitches in the future where someone edits the command but not what gets `echo`ed?
```bash
cmd="insert command here"
echo -e "--- ${cmd}"
eval "${cmd}"
```

Reply:
@jmgate I'll have a look. Usually in Bash what I like to do is something like this:

```bash
cmd="the_executable"
args=(
    arg1
    arg2
    arg3
)
echo -e "--- ${cmd} ${args[@]}"
eval ${cmd} ${args[@]}
```

I like the associative arrays because if you want to disable one of the arguments you can just comment out that line in the array.  When people use continuation lines for long commands like this:

```bash
mycommand \
    arg1 \
    arg2 \
    arg3
```
if you want to turn off arg2, you can't just comment out that line because it will cause an error in bash.

In the past, I haven't typically launched other scripts via `eval`... is there any advantage to using that versus just launching the command?

Reply:
I will note though, in this case I was just cutting and pasting from the info I got from Python's website.  We're going to be removing the Python2 dependence on our PR driver scripts either as part of this update or right after we get this in... Since the 'get-pip.py' script need is mostly for Python2 support since our Python3 implementations actually have pip3 packaged with them this little function will be able to be removed.  

Many things get easier once we can ditch python2 for PR testing purposes... it's end-of-life was Jan 1, 2020 anyhow so pretty much everyone ought to be phasing out Python2 use anyhow ;)

Reply:
I've made this change in my working copy, it'll be in with the next push.